test/perf.js

```diff
@@ -103,6 +103,9 @@ for (var i = 0; i < N; i++) {
 }
 console.log('JSON.stringify(JSON.parse())', N / (Date.now() - start))
 
+var b = Buffer.alloc(BIPF.encodingLength(pkg))
+BIPF.encode(pkg, b, 0)
```

Comment:
I did this here because all the tests below assume that package.json fields exist, while `fakeData` would never have those fields.